shared/message/motion/FootMotionCommand.proto

```diff
@@ -1,57 +0,0 @@
-/*
```

Comment:
MIght not be a wise idea to delete this file just yet?

Reply:
I think this was just used by the deleted modular walk engine